src/sources/datadog_agent/mod.rs

```diff
@@ -194,51 +195,44 @@ impl SourceConfig for DatadogAgentConfig {
             .decoding
             .schema_definition(global_log_namespace.merge(self.log_namespace))
             .with_source_metadata(
-                self.get_component_name(),
-                Some("message"),
-                "message",
-                Kind::bytes(),
-                Some("message"),
```

Comment:
Is the dd decoder properly handling this type definition?

Reply:
Yes, this is handled by the decoder.